AdminWebsite/AdminWebsite/ClientApp/src/app/services/bookings-persist.service.spec.ts

```diff
@@ -136,6 +136,30 @@ describe('BookingsPersistService', () => {
         });
     });
 
+    describe('#noAllocated', () => {
+        it('should set noAllocatedHearings', () => {
```

Comment:
these tests don't offer much other than assert on a value you set yourself. There should be a behaviour change or something (i.e. a function call) between the arrange and assert

Reply:
Is there anything in the `bookings-persist.service.ts` that manipulate these fields that can be called as a part of the test?